src/ui-test/tests/editor.actions.test.ts

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
+import * as path from 'path';
+import {
+    after,
+    before,
+    By,
+    EditorView,
+    VSBrowser,
+    WebElement,
+} from 'vscode-extension-tester';
+
+describe('Camel file editor test', function () {
+
+    const CAMEL_RUN_DEBUG_ACTION_LABEL = 'Start Camel Application with JBang and Debug';
+    const CAMEL_ROUTE_YAML = 'demo route.camel.yaml';
+
+    describe('Camel Actions', function () {
+        this.timeout(30000);
+
+        before('Before setup', async function () {
+            await VSBrowser.instance.openResources(path.resolve('src', 'ui-test', 'resources', CAMEL_ROUTE_YAML));
+            await VSBrowser.instance.waitForWorkbench();
+        });
+
+        after('After cleanup', async function () {
+            await new EditorView().closeAllEditors();
+        });
+
+        it('Debug and Run action is available', async function () {
+            const button = await getAction(CAMEL_RUN_DEBUG_ACTION_LABEL);
+            expect(button).to.not.undefined;
+        });
+
+    });
+
+    async function getAction(title: string): Promise<WebElement | undefined> {
```

Comment:
spotted bug in extester for vscode 1.76+ there is broken EditorView.getAction() method --> I will report on vscode-extension-tester and open JIRA to remove this workaround from our uitests after new extester release with fix

Reply:
ExTester upstream bug - https://github.com/redhat-developer/vscode-extension-tester/issues/724